packages/plugin/src/theia.d.ts

```diff
@@ -7231,25 +7231,28 @@ declare module '@theia/plugin' {
          * @param uri A resource identifier.
          * @param range A range.
          * @param newText A string.
+         * @param metadata Optional metadata for the entry.
          */
-        replace(uri: Uri, range: Range, newText: string): void;
+        replace(uri: Uri, range: Range, newText: string, metadata?: WorkspaceEditMetadata): void;
```

Comment:
I didn't find 
`export interface WorkspaceEditMetadata`
within `theia.d.ts`

btw: I see that [VS Code uses](https://github.com/microsoft/vscode/blob/14767349ab8b56cdff3c015b6a7c6d24831c387d/src/vs/vscode.d.ts#L3070) WorkspaceEdit**Entry**Metadata

Reply:
fixed